system/services/validation/PresideFieldRuleGenerator.cfc

```diff
@@ -86,8 +86,17 @@ component output="false" singleton=true {
 
 					if ( StructKeyExists( field, "rules" ) ) {
 						for( rule in field.rules ){
-							rule.fieldName  = field.name;
-							rule.fieldLabel = field.label;
+							rule.fieldName        = field.name;
+							rule.fieldLabel       = field.label;
+							rule.paramsFieldLabel = {};
+							if ( !StructIsEmpty( rule.params ?: {} ) && StructKeyExists( rule.params, "field" ) ) {
+								for ( var f in fieldset.fields ) {
```

Comment:
Will this only work if the referenced field is in the same fieldset?

Reply:
Hi @sebduggan ,

Yup, it would only work if the field is in the same fieldsets as I have not seen reference field to be on different fieldset. 
I could update it to look through all the fieldsets, but wondering if we should also look through all the tabs, tho I doubt reference field going to be defined as such.

Let me know your thought.